components/person/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { PersonRecord } from 'types'
+import Image from 'next/image'
+import style from './style.module.css'
+export interface PeopleProps {
+	updatedList: PersonRecord[]
+}
+
+export interface AvatarProps {
+	person: PersonRecord
+}
+function Avatar({ person }: AvatarProps) {
+	if (!person.avatar || person.avatar === null) {
+		return (
+			<Image
+				className={style.avatarImage}
+				src="/profileAlt.png"
+				alt="no image for user"
+				width={105}
+				height={109}
+			/>
+		)
+	}
+	if (!person.avatar.alt) {
+		return (
+			<Image
+				className={style.avatarImage}
+				src={person.avatar.url}
+				alt="no alt name for user"
+				width={105}
+				height={109}
+			/>
+		)
+	}
+	return (
+		<Image
+			className={style.avatarImage}
+			src={person.avatar.url}
+			alt={person.avatar.alt}
+			width={105}
+			height={109}
+		/>
+	)
+}
+
+export default function Person({
+	updatedList,
+}: PeopleProps): React.ReactElement {
+	return (
+		<div className={style.peopleSection}>
+			{updatedList.length > 0 ? (
+				updatedList.map((person: PersonRecord) => (
+					<div key={person.id} className={style.peopleCube}>
+						<Avatar person={person}></Avatar>
+						<div className={style.personInfo}>
+							<ul className={style.list}>
+								<li className={style.nameType}>{person.name}</li>
+								<li className={style.titleType}>{person.title}</li>
+								<li className={style.departmentType}>
+									{person.department.name}
+								</li>
+							</ul>
+						</div>
+					</div>
+				))
+			) : (
+				<div>
+					<p>no results found</p>
```

Comment:
```suggestion
					<p>No results found</p>
```
<img width="1017" alt="Screenshot 2023-10-10 at 1 52 02 PM" src="https://github.com/kristen-kagei/mktg-webdev-challenge/assets/36613477/2488b967-e383-485f-a1aa-7b65cff4d96a">

Noting that this default should have some styles applied.